network/unstaked/conduit.go

```diff
@@ -0,0 +1,26 @@
+package unstaked
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/network"
+)
+
+// UnstakedConduit is a special conduit used by unstaked nodes which wraps
+// the default conduit implementation and replaces the target of every network
+// send with the upstream staked node.
+type UnstakedConduit struct {
+	network.Conduit
+	stakedNodeID flow.Identifier
+}
+
+func (c *UnstakedConduit) Publish(event interface{}, targetIDs ...flow.Identifier) error {
+	return c.Conduit.Unicast(event, c.stakedNodeID)
```

Comment:
so it will be better if we still use `Publish` here rather than `Unicast`. Unicast will create a dedicated connection between the unstaked access node and the staked AN just for this request. A `Publish` will send this message over libp2p pub-sub mesh and can hop through intermediate nodes (gossip).